server/main.go

```diff
@@ -34,16 +34,18 @@ var (
 )
 
 type Server struct {
-	BindAddress  string `json:"bind_address,omitempty"`
-	ListenPort   int    `json:"listen_port,omitempty"`
-	MetricsPort  int    `json:"metrics_port,omitempty"`
-	MaxTokens    int    `json:"max_tokens,omitempty"`
-	KeyFilePath  string `json:"key_file_path"`
-	CommFilePath string `json:"comm_file_path"`
-
-	key []byte        // a big-endian marshaled big.Int representing an elliptic curve scalar
-	G   *crypto.Point // elliptic curve point representation of generator G
-	H   *crypto.Point // elliptic curve point representation of commitment H to key
+	BindAddress        string `json:"bind_address,omitempty"`
+	ListenPort         int    `json:"listen_port,omitempty"`
+	MetricsPort        int    `json:"metrics_port,omitempty"`
+	MaxTokens          int    `json:"max_tokens,omitempty"`
+	SignKeyFilePath    string `json:"sign_key_file_path"`
```

Comment:
this breaks backward compatibility, so we should be careful when upgrading after this change.

Reply:
good point, I could signing key id back to the original one? The extra keys for redemption are optional anyway so this would mean there would have to be little change to the deployment.